src/devtools/client/debugger/src/components/shared/ResultList.tsx

```diff
@@ -6,24 +6,51 @@ import classnames from "classnames";
 //
 import React, { Component } from "react";
 
+import { scrollList } from "../../utils/result-list";
+import { SearchResultWithHighlighting } from "../QuickOpenModal";
 import AccessibleImage from "./AccessibleImage";
 
-export default class ResultList extends Component {
+interface ResultListProps {
+  dataTestId?: string;
+  role?: string;
+  items: SearchResultWithHighlighting[];
+  expanded: boolean;
+  selected: number;
+  size?: string;
+  selectItem: (e: any, item: SearchResultWithHighlighting) => void;
+}
+
+export default class ResultList extends Component<ResultListProps> {
+  listItemRefs: Record<number, HTMLElement> = {};
```

Comment:
Subjective nit: This is kind of confusingly named because it's not a map of _refs_ but a map of actual HTML elements.

I guess those _are_ "refs" but I think of a "ref" as a thing with a `.current` property.